tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -544,13 +546,13 @@ async def test_tls_exchange_replicas_replicas_with_ror(self, bft_network):
         for i in range(301): # Produce 301 new blocks
             await skvbc.send_write_kv_set()
         # Now, lets switch keys to all replicas
-        exchanged_replicas = bft_network.all_replicas()
+        exchanged_replicas = bft_network.all_replicas(without={1,2,3,4,5,6})
```

Comment:
can you explain this change? why do we have to exchange the TLS keys only for replica 0 now?

Reply:
Reverted